dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -130,29 +135,25 @@ public CompletableResultCode export(@Nonnull Collection<MetricData> metrics) {
       return CompletableResultCode.ofFailure();
     }
 
-    return export(metrics, connection);
+    return doExport(metrics, connection);
   }
 
-  private List<String> serializeToMetricLines(Collection<MetricData> metrics) {
+  @VisibleForTesting
```

Comment:
It doesn't give me warnings. Its not needed, in fact its just a "you should not use this" since we cannot enforce it, but I think we should test this functionality.